platform/admin/core/admin-core-appservice/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -11,20 +11,15 @@
  *
  **/ -->
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
-           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"
            xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.0.0"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
     <ext:property-placeholder/>
 
-    <bean id="appService"
-          class="org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl">
+    <bean id="appService" class="org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl">
         <cm:managed-properties
             persistent-id="org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl"
             update-strategy="container-managed"/>
-        <argument ref="bundleStateServices"/>
-        <property name="configFileName"
-                  value="${ddf.home}/etc/org.codice.ddf.admin.applicationlist.properties"/>
```

Comment:
❓Since your getting rid of applicationlist.properties can we now accomplish the same thing by making our apps boot features?

Reply:
Yeah, you should be able to now since all the bundles to display in the admin console are now defined in the app feature. Although, I question how stable it would be to start up app features independently right now. There's still a lot of missing dependencies in app features that are only working because of the profiles starting up more than one app.

Reply:
hmm, can we leave this functionality in then? or will another PR be addressing the dependancies shortly? I would like to still be able to skip the install screen and just start all the features my distribution needs on startup.

Reply:
You could always put a profile as a boot feature, or define your own boot profile that points to the features you need. I'd prefer to completely decouple app from features if possible.

Reply:
:question: Did you actually remove https://github.com/codice/ddf/blob/master/distribution/ddf/src/main/filtered-resources/etc/org.codice.ddf.admin.applicationlist.properties?